### PR TITLE
Fixes to Python docs

### PR DESCRIPTION
**Pull request by jamesbraza**
- Fixes missing syntax (e.g. closing paren, colon)
- Removes extra `else`
- Adds missing imports

### CODE REVIEW

File: site/docs/configuration/expected-outputs/python.md [hunk 1]
```diff
@@ -39,19 +39,20 @@ assert:
...
 
 ## Using test context
 
 A `context` object is available in the Python function. Here is its type definition:
 
 ```py
-class AssertContext:
+from typing import Dict, TypedDict, Union
+
+class AssertContext(TypedDict):
```
Comment (jamesbraza) [inline site/docs/configuration/expected-outputs/python.md:55]: Shouldn't `vars` allow for `dict` too? Was that what was meant by `object`?

Comment (jamesbraza) [reply site/docs/configuration/expected-outputs/python.md:55]: Also, it looks like `test` (also a `dict`) should be here too

Comment (typpo) [reply site/docs/configuration/expected-outputs/python.md:55]: Yeah, I think you're right on both counts

Comment (jamesbraza) [reply site/docs/configuration/expected-outputs/python.md:55]: Okay, implemented them, this is ready for review now

**Comment by typpo** [comment]
Thanks James!

**Pull request merged**